### PR TITLE
Updates the subject lines for peer endorsement emails

### LINKED ISSUE #822: Wording of peer endorsements confirmation emails is not consistent
The style and wording of the peer endorsements confirmation emails is not consistent.

### PR DESCRIPTION
**Pull request by dakota002**
Closes #822 

@lpsinger and @jracusin any suggestions for the wording of the email bodies?

### CODE REVIEW

**Comment by lpsinger** [comment]
- [x] Subject lines should be consistent: they should all start with "GCN Peer Endorsements".
- [ ] All of the email bodies should tell the user what their next action should be. The new endorsement email does a great job of this, but we need to explain what the user is supposed to do if their endorsement was approved or rejected.
- [ ] All of the emails should contain a link to the endorsements page.
- [ ] The state change emails need more descriptive titles. Right now they are all merely "GCN Peer Endorsements: Status Update".
- [ ] Use consistent tenses and avoid passive voice.

File: app/routes/user/endorsements.server.ts [hunk 1]
```diff
@@ -205,38 +206,52 @@ export class EndorsementsServer {
...
+    if (status === 'approved') {
       promiseArray.push(
         this.#addUserToGroup(requestorSub),
         clearUserToken(requestorSub)
       )
-
-    if (status === 'reported')
+      requestorMessage += `
+      
+      As an approved user, you may submit GCN Circulars at ${origin}/circulars/new and be requested for endorsement by other users.
```
Comment (lpsinger) [inline app/routes/user/endorsements.server.ts:220]:
```suggestion
      As an approved user, you may now submit GCN Circulars at ${origin}/circulars/new and be requested for endorsement by other users.
```

File: app/routes/user/endorsements.server.ts [hunk 2]
```diff
@@ -205,38 +206,52 @@ export class EndorsementsServer {
...
         fromName,
         to: [this.#currentUserEmail],
-        subject: 'GCN Peer Endorsement Status Update',
-        body: `Your changes to ${requestorEmail}'s peer endorsement request have been processed. They will receive an email as well to confirm the new status.`,
+        subject: `GCN Peer Endorsements: Endorsement ${status}`,
+        body: dedent`Your changes to ${requestorEmail}'s peer endorsement request have been processed. They will receive an email as well to confirm the new status.
+        
+        No further action is required on your part for this user's request.
+        
+        Review your remaining endorsement requests here: ${origin}/user/endorsements`,
```
Comment (lpsinger) [inline app/routes/user/endorsements.server.ts:254]:
```suggestion
        Review your endorsement requests here: ${origin}/user/endorsements`,
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/user/endorsements.server.ts [hunk 3]
```diff
@@ -205,38 +206,52 @@ export class EndorsementsServer {
...
         this.#addUserToGroup(requestorSub),
         clearUserToken(requestorSub)
       )
-
-    if (status === 'reported')
+      requestorMessage += `
+      
+      As an approved user, you may now submit GCN Circulars at ${origin}/circulars/new and be requested for endorsement by other users.
+      
+      View your pending endorsements here: ${origin}/user/endorsements`
```
Comment (lpsinger) [inline app/routes/user/endorsements.server.ts:222]: This last line of the text with the link to the endorsements page should be consistent across different emails too.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/user/endorsements.server.ts (hunk 3 above)
Comment (dakota002) [reply app/routes/user/endorsements.server.ts:222]: Are you thinking `View your pending endorsements here: ${origin}/user/endorsements` for all of them, or something that sounds like it applies to either endorser or requester?

Comment (lpsinger) [reply app/routes/user/endorsements.server.ts:222]: I think it applies to either.

**Review by lpsinger — APPROVED** [review]

**Pull request merged**